assets/javascript/gallery.js

```diff
@@ -0,0 +1,10 @@
+$(function(){
```

Comment:
Then preprocess the JSON result to make it accessible to Go. Keep the preprocessed result in memory and don't write anything to disk.